egs2/aishell/asr2/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+
+kmeans_feature="wavlm_large/21"  # use model_type/layer_index
```

Comment:
Just a note, `wavlm_large` was trained on EN data. Maybe it doesn't work well on CN speech data.

Reply:
Yeah, I am curious whether the discrete units are language dependent. I will test Chinese as well as multilingual upstream models later.